From 474d6099a1b1c4de490df68bd427f1371e3d4337 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Wed, 11 Dec 2019 16:22:24 +0100
Subject: [PATCH 10/12] meson: Add the libhandy 0.0.12 dep

This will allows to access many adaptive widgets to port the shell to
phones.
---
 meson.build       | 1 +
 shell/meson.build | 1 +
 2 files changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index aa0fe96d..1b751c52 100644
--- a/meson.build
+++ b/meson.build
@@ -155,6 +155,7 @@ gmodule_dep = dependency('gmodule-2.0')
 gmodule_no_export_dep = dependency('gmodule-no-export-2.0', version: glib_req_version)
 gtk_dep = dependency('gtk+-3.0', version: gtk_req_version)
 gthread_dep = dependency('gthread-2.0', version: glib_req_version)
+libhandy_dep = dependency('libhandy-0.0', version: '>= 0.0.12')
 
 m_dep = cc.find_library('m')
 
diff --git a/shell/meson.build b/shell/meson.build
index 7d76839c..0c59d85e 100644
--- a/shell/meson.build
+++ b/shell/meson.build
@@ -70,6 +70,7 @@ deps = [
   libdazzle_dep,
   libgd_dep,
   libgimpcellrenderertoggle_dep,
+  libhandy_dep,
   m_dep,
 ]
 
-- 
2.30.1

